.circleci/config.yml

```diff
@@ -194,6 +196,7 @@ workflows:
             branches:
               only:
                 - master
+                - build-fix
```

Comment:
Should remove this before merge IMO, but the rest looks good to me! (contingent on deploy working with migrations changed)

Reply:
Update: Build seems to still be broken, an issue with docker-compose

Reply:
Tentative approval once `docker-compose` gets up 🚀